src/icon_exclaim_perf_tools/log_import.py

```diff
@@ -177,6 +177,8 @@ def import_timer_report(
     # skip header
     header_dash_pattern = re.compile("([-]+)".join(["(\\s+)"] * (len(columns.values())+1)))
     line_iterator.skip("")
+    while re.match(r"mo_.*:.*", line_iterator.current_line().strip()):
```

Comment:
```suggestion
    # TODO: 
    while re.match(r"^[\w_:]+: (start|end)$", line_iterator.current_line().strip()):
```
Little more generic with respect to the "word" characters and more restrictive in format. Not sure if it's `end` or `stop`. Important part is `^$`.